test/unit/mock/profiles/unsupported_dependencies/vendor/e39eb85366b272bae98e5eecdfac9f84c50a9ae9dd625fba2ce847268a6c3477/controls/profile1.rb

```diff
@@ -0,0 +1,12 @@
+control 'require-01' do
```

Comment:
Comments here explaining how this fixture is intended to be used would be helpful.